tests/expectations/test_expectation_arguments.py

```diff
@@ -582,15 +663,55 @@ def test_result_format_configured_with_set_default_override(
 
     # Test calling "validator.validate()" explicitly.
 
-    validator_validation: ExpectationSuiteValidationResult = validator.validate()
-    results: List[ExpectationValidationResult] = validator_validation.results
+    validator_validation: ExpectationSuiteValidationResult
+
+    validator_validation = validator.validate()
+
+    results: List[ExpectationValidationResult]
+
+    results = validator_validation.results
     assert len(results) == 1
 
     result: ExpectationValidationResult
 
     result = results[0]
     assert len(result.result.keys()) == 0
 
+    result_format = "BASIC"
+    runtime_environment_arguments: dict = {
+        "catch_exceptions": catch_exceptions,
+        "result_format": result_format,
+    }
+
+    suite = in_memory_runtime_context.create_expectation_suite(
+        "test_suite", overwrite_existing=True
+    )
+
+    expectation_arguments_without_meta = dict(
+        **runtime_environment_arguments, **expectation_arguments_column
+    )
+    expectation_configuration = ExpectationConfiguration(
+        expectation_type="expect_column_values_to_not_be_null",
+        kwargs=expectation_arguments_without_meta,
+        meta=expectation_meta,
+    )
+    suite.add_expectation(expectation_configuration=expectation_configuration)
+
+    validator = in_memory_runtime_context.get_validator(
+        batch_request=runtime_batch_request,
+        expectation_suite=suite,
+    )
+
+    validator.set_default_expectation_argument("result_format", "BOOLEAN_ONLY")
+
+    validator_validation = validator.validate()
+
+    results = validator_validation.results
+    assert len(results) == 1
+
+    result = results[0]
+    assert len(result.result.keys()) == 0
+
```

Comment:
For the various `result_format` tests, please add a literal check for the result object - a comparison between the expected and actual ExpectationValidationResults.result object, so that it is clear from the test how each object is returned.

Reply:
Done.